tests/migration/migration.go

```diff
@@ -645,12 +645,11 @@ var _ = SIGMigrationDescribe("VM Live Migration", func() {
 				errorMessageFormat := "failed after running `kill -9 %v` with stdout:\n %v \n stderr:\n %v \n err: \n %v \n"
 				Expect(err).ToNot(HaveOccurred(), fmt.Sprintf(errorMessageFormat, pid, stdout, stderr, err))
 
-				// wait for both virtqemud to respawn and all connections to re-establish
-				time.Sleep(30 * time.Second)
-
 				// ensure new pid comes online
-				newPid := getVirtqemudPid(&pods.Items[0])
-				Expect(pid).ToNot(Equal(newPid), fmt.Sprintf("expected virtqemud to be cycled. original pid %s new pid %s", pid, newPid))
+				Eventually(getVirtqemudPid).WithArguments(pod).
+					WithPolling(5*time.Second).
```

Comment:
Can we be more aggressive than 5 seconds or is it not worth?

Reply:
I think we can and we **must**. Currently, the timeout is only 10 seconds and hence there will be probably only 1 try.
I also increased the timeout, to be aligned with the previously 30 seconds of sleep.